troubleshooting/known_issues/cray_console_node_in_CLBO_state.md

```diff
@@ -0,0 +1,79 @@
+# `CRAY-CONSOLE-NODE` pods intermittently goes into CLBO state after fresh CSM install
+
+## Issue Description
+
+After installing CSM 1.6, `CRAY-CONSOLE-NODE` pods intermittently goes into CLBO state. This happens because of 
```

Comment:
Is this actually intermittent? As in, will it recover on its own? Or is it something that will keep happening once you encounter it?